meta-lmp-base/recipes-security/optee/optee-os-tadevkit_3.17.0.bb

```diff
@@ -1,12 +1,17 @@
+DEFAULT_PREFERENCE ="${@bb.utils.contains('PREFERRED_PROVIDER_virtual/optee-os', 'optee-os', '-1', '0', d)}"
```

Comment:
Add extra space after '='.

Reply:
Please also add a small comment line saying why (basically because tadevkit requires a matching version on the base recipe, and meta-arm kirkstone is using 3.16 by default).